src/pages/Home.jsx

```diff
@@ -10,13 +33,9 @@ const Home = () => {
         </IonToolbar>
       </IonHeader>
       <IonContent className="ion-padding">
-        <p>
-          This is the  best mobile news app.{' '}
-          <a target="_blank" rel="noopener noreferrer" href="https://ionicframework.com/docs/">
-            Subscribe Here!
-          </a>{' '}
-          WELCOME!
-        </p>
+        <IonList>
+          {articleItems}
```

Comment:
you don't want to show the truncated body of the article?

Reply:
https://recordit.co/sfswniqWTb

Reply:
yes. with this display i could do it without using trauncate.